packages/windows_security/lib/src/winrt_com_interop_helpers.dart

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2023, Dart | Windows. Please see the AUTHORS file for details.
+// All rights reserved. Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Helper wrapper classes for WinRT COM interop interfaces.
+// See https://learn.microsoft.com/en-us/windows/apps/develop/ui-input/display-ui-objects
+
+import 'dart:async';
+import 'dart:ffi';
+
+import 'package:ffi/ffi.dart';
+import 'package:win32/win32.dart';
+import 'package:windows_foundation/internal.dart';
+import 'package:windows_foundation/windows_foundation.dart';
+
+import 'authentication/web/core/webtokenrequest.dart';
+import 'authentication/web/core/webtokenrequestresult.dart';
+import 'credentials/webaccount.dart';
+
+/// Provides Win32 apps with access to certain functions of
+/// `WebAuthenticationCoreManager` that are otherwise available only to UWP
+/// apps.
+///
+/// {@category class}
+final class WebAuthenticationCoreManagerInterop {
+  static const _className =
+      'Windows.Security.Authentication.Web.Core.WebAuthenticationCoreManager';
+  // IID of IAsyncOperation<WebTokenRequestResult>
+  static const _iid = '{0a815852-7c44-5674-b3d2-fa2e4c1e46c9}';
+
+  /// Asynchronously requests a token from a web account provider.
+  /// If necessary, the user is prompted to enter their credentials.
+  ///
+  /// [appWindow] represents the window to be used as the owner for the window
+  /// prompting the user for credentials, in case such a window becomes
+  /// necessary. If omitted, [getWindowHandle] is used.
```

Comment:
I wonder if we should describe what `getWindowHandle` does, rather than point to that function? 

```suggestion
/// necessary. If omitted, the current console or shell's desktop window is used.
```

Reply:
Much better, thank you.